app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class CardsMailer < ActionMailer::Base
+  default from: 'info@flashcards.com'
+
+  def pending_cards_notification(email, cards_count)
```

Comment:
Unused method argument - `cards_count`. If it's necessary, use `_` or `_cards_count` as an argument name to indicate that it won't be used.